packages/core/src/promotionEvaluations/redux/selectors.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @module promotionEvaluations/selectors
+ * @category Promotion Evaluations
+ * @subcategory Selectors
+ */
+
+import { getError, getIsLoading, getResult } from './reducer';
+
+/**
+ * Retrieves a list of all the promotion evaluations items available.
+ *
+ * @function
+ * @param {object} state - Application state.
+ * @returns {Array} - List of promotion evaluations items.
+ * @example
+ * import { getPromotionEvaluationItems } from '@farfetch/blackout-core/promotionEvaluations/redux';
+ *
+ * const mapStateToProps = state => ({
+ * promotionEvaluationsItems: getPromotionEvaluationItems(state)
```

Comment:
```suggestion
 *   promotionEvaluationsItems: getPromotionEvaluationItems(state)
```

Applicable everywhere 🙂

Reply:
Done 🙂